website/docs/docs/use-dbt-semantic-layer/gsheets.md

```diff
@@ -53,18 +48,22 @@ The Google Sheets **Query Builder** custom menu has the following capabilities:
 | Menu items    | Description                                           |
 |---------------|-------------------------------------------------------|
 | Metrics       | Search and select metrics.                             |
-| Group By      | Search and select dimensions or entities to group by. Dimensions are grouped by the entity of the semantic model they come from. |
-| Time Range    | Quickly select time ranges to look at the data, which applies to the main time series for the metrics (metric time). You can still optionally use `where` to filter time ranges|
+| Group By      | Search and select dimensions or entities to group by. Dimensions are grouped by the entity of the semantic model they come from. You may choose dimensions on their own without metrics. |
+| Time Range    | Quickly select time ranges to look at the data, which applies to the main time series for the metrics (metric time), or do more advanced filter using the "Custom" selection|
 | Granularity   | Modify the granularity of the primary time dimension.      |
 | Where         | Filter your data. This includes categorical and time filters. |
 | Order By      | Return your data order.                              |
 | Limit         | Set a limit for the rows of your output.               |
 
+
+
 **Filtering data**
 
 To use the filter functionality, choose the [dimension](docs/build/dimensions) you want to filter by and select the operation you want to filter on.
-- For categorical dimensions, type in the dimension value you want to filter by (no quotes needed) and press enter. 
-- Continue adding additional filters as needed with AND and OR. If it's a time dimension, choose the operator and select from the calendar.
+- For categorical dimensions, you can type a value into search or select from a populated list. For entities, you must type the value you are looking for as we do not load all of them given the large number of values.
+- Continue adding additional filters as needed with AND and OR.
+
+For time dimensions, you can use the time range selector to filter on the primary time dimension with preset or custom options. For non-primary time dimensions, use the "Where" option to apply filters.
```

Comment:
```suggestion
For time dimensions, you can use the time range selector to filter on the primary time dimension with preset or custom options. For non-primary time dimensions series for metrics, use the "Where" option to apply filters.
```

Reply:
should we say 'For non-metric time dimensions, use the "where"...'?

Reply:
Ok I tweaked a bit! let me know your thoughts